src/test/kotlin/org/jetbrains/plugins/template/MyPluginTest.kt

```diff
@@ -0,0 +1,30 @@
+package org.jetbrains.plugins.template
+
+import com.intellij.ide.highlighter.XmlFileType
+import com.intellij.psi.xml.XmlFile
+import com.intellij.testFramework.fixtures.BasePlatformTestCase
+import com.intellij.util.PsiErrorElementUtil
+
+class MyPluginTest : BasePlatformTestCase() {
+
+    fun `test created XML file`() {
+        val psiFile = myFixture.configureByText(XmlFileType.INSTANCE, "<foo>bar</foo>")
+        assertInstanceOf(psiFile, XmlFile::class.java)
+
+        assertFalse(PsiErrorElementUtil.hasErrors(project, psiFile.virtualFile))
+
+        psiFile as XmlFile
+        assertNotNull(psiFile.rootTag)
+
+        psiFile.rootTag?.let {
+            assertEquals("foo", it.name)
+            assertEquals("bar", it.value.text)
+        }
+    }
+
+    override fun getTestDataPath() = "src/test/resources/rename"
```

Comment:
https://plugins.jetbrains.com/docs/intellij/test-project-and-testdata-directories.html
we usually recommend having dedicated `testData` folder separate from actual _resources_ stored in test/resources